incubator/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,21 +0,0 @@
-name: fluentd-elasticsearch
```

Comment:
was it intentional to delete the incubator-files?

Reply:
See https://github.com/helm/charts/blob/master/PROCESSES.md for how that works

Reply:
@davidkarlsen the process does not explain when a deprecated chart should be removed. I thought when a chart is promoted as stable the deprecated one could be removed. Anyway, I reverted my last commit. ty